pyproject.toml

```diff
@@ -142,6 +144,9 @@ gitpython = "==3.1.14"
 ipfshttpclient = "==0.8.0a2"
 
 
+[tool.poetry.group.types]
```

Comment:
Realised this is missing (unlike other groups) for the `types` group. 
@solarw Is it needed?

Reply:
yesm it's a good point. let stay it optional